### PR TITLE
chore: ui amends based on feedback

### LINKED ISSUE #91: Update text and link color
Text & Links are using violet-900. Needs to be updated to accent (#3E238B)

Current
Screenshot 2023-09-06 at 3 42 30 PM

Expected
Screenshot 2023-09-06 at 3 43 11 PM

### PR DESCRIPTION
**Pull request by ShaunBaker**
resolves #91 #92 #104 #97 #96 #93 #102 #98 #94 #100 #95 and #99

### CODE REVIEW

File: src/routes/home.tsx [hunk 1]
```diff
@@ -83,8 +83,15 @@ export default function HomeRoute() {
...
-                      size="sm"
+                      label={
+                        <p>
+                          Sort by{' '}
+                          <span className="text-sm font-normal leading-tight pl-1">
+                            {sort}
+                          </span>
+                        </p>
+                      }
+                      size="xs"
```
Comment (jack-robson) [inline src/routes/home.tsx:94]: I think  `<p>Sort by{' '}` should exist outside <Dropdown> to better match the intent of #95.

Same for the Show Dropdown.

File: src/components/layout/header.tsx
Comment (jack-robson) [inline src/components/layout/header.tsx:1]: Header.tsx is a bit hard to read, maybe move out the SVG to their own files should make it a bit easier.

**Review by jack-robson — CHANGES_REQUESTED** [review]
Looking good only a couple small comments.

Tested all the issues and seem's to fix them all except #95 needs a little tweak.

**Comment by simoneb** [comment]
@ShaunBaker can you please review @jack-robson 's feedback?

**Review by jack-robson — APPROVED** [review]

**Pull request merged**